### ISSUE TITLE
App built for iOS 14 fails to load the Razorpay checkout page

### ISSUE THREAD

**Issue by ebizbuddy** [razorpay/razorpay-capacitor#29]
App built with Ionic 5, Capacitor 3.2.2, Angular 12.2.4, @ionic/angular 5.7.0

This razorpay-capacitor plugin installed using:
npm i -S https://github.com/razorpay/razorpay-capacitor.git --legacy-peer-deps

**App deployed _on iPhone (iOS 14.7.1)_ is unable to load the Razorpay checkout page.**

Error:
Unhandled Promise Rejection: DataCloneError: The object can not be cloned.
At:
nativeBridge -> initBridge -> function initNativeBridge(win) -> postToNative -> win.webkit.messageHandlers.bridge.postMessage(data);

Please check and suggest a solution.

PS: Razorpay checkout page in an Android App works without any error.

***************************************************************************
Environment details:

Ionic:

   Ionic CLI                     : 6.17.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.7.0
   @angular-devkit/build-angular : 12.2.4
   @angular-devkit/schematics    : 12.2.4
   @angular/cli                  : 12.2.4
   @ionic/angular-toolkit        : 4.0.0

Capacitor:

   Capacitor CLI      : 3.2.2
   @capacitor/android : 3.2.2
   @capacitor/core    : 3.2.2
   @capacitor/ios     : 3.2.2

Utility:

   cordova-res : 0.15.3
   native-run  : 1.4.1

System:

   NodeJS : v14.17.5 (/usr/local/bin/node)
   npm    : 7.22.0
   OS     : macOS Big Sur - Mac Mini (2018)
   Processor : 9th Gen Intel Core i7

***************************************************************************

**Comment by vivekshindhe** [comment]
@ebizbuddy Could you please help with the error that you are facing when you do ``npm install --save https://github.com/razorpay/razorpay-capacitor.git``

**Comment by ebizbuddy** [comment]
> @ebizbuddy Could you please help with the error that you are facing when you do `npm install --save https://github.com/razorpay/razorpay-capacitor.git`

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: netpal@2.1.0
npm ERR! Found: @capacitor/core@3.2.3
npm ERR! node_modules/@capacitor/core
npm ERR!   @capacitor/core@"^3.2.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @capacitor/core@"^2.4.6" from capacitor-razorpay@0.0.1
npm ERR! node_modules/capacitor-razorpay
npm ERR!   capacitor-razorpay@"https://github.com/razorpay/razorpay-capacitor.git" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/dev3/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dev3/.npm/_logs/2021-09-21T07_28_27_343Z-debug.log

I used --legacy-peer-deps to bypass this error.
